website_and_docs/content/documentation/webdriver/elements/information.zh-cn.md

```diff
@@ -343,10 +384,9 @@ val text = driver.findElement(By.cssSelector("h1")).getText()
 {{< /tab >}}
 {{< /tabpane >}}
 
-## 获取特性或属性
+## 获取属性
```

Comment:
Can you diff the attribute and property? The reader will be confused if you did as above way